### PR TITLE
Release/0.6.4

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: provisioning/resources/elasticsearch/config/kibana.yml [hunk 1]
```diff
@@ -10,13 +10,13 @@ server.host: "0"
 # Use the `server.rewriteBasePath` setting to tell Kibana if it should remove the basePath
 # from requests it receives, and to prevent a deprecation warning at startup.
 # This setting cannot end in a slash.
-server.basePath: "kibana"
+server.basePath: "/kibana"
```
Comment (jbeemster) [inline provisioning/resources/elasticsearch/config/kibana.yml]: Is the forward slash required?

Comment (oguzhanunlu) [reply provisioning/resources/elasticsearch/config/kibana.yml]: yes, I tested now and it doesn't mount at /kibana otherwise

Comment (jbeemster) [reply provisioning/resources/elasticsearch/config/kibana.yml]: Cool thanks for checking!

**Comment by oguzhanunlu** [comment]
Squashed commits and added release commit after Josh's approval, hence force-push.

**Pull request merged**